reorg of staking technical docs (#142)
onflow/flow: docs/content/token/staking/stake-slashing.mdx

---
title: Stake Slashing

> description: How the Flow protocol punishes bad behavior be slashing stakers tokens

Reviewer: ```suggestion
description: How Flow enforces honest node behaviour
```

Feel free to ignore or reword this suggestion. The description field is used in link previews and therefore is often truncated, so I try to keep them a bit shorter

---

## Slashing

Severe slashing on Flow only takes place in the most nefarious of attacks. 
The protocol reserves slashing for maintaining the security of the protocol rather than its liveness.

> You can find more details on the conditions under which a node is slashed in the Flow white papers.

Reviewer: ```suggestion
You can find more details on the conditions under which a node is slashed in the Flow whitepapers.
```

> While a node is in operation and during the unstaking period after they request unstaking.

Reviewer: ```suggestion
While a node is in operation and during the unstaking period after they request unstaking,
```

the staked tokens are held by the staking smart contract.
Slashable protocol violations must be adjudicated by a supermajority 
of more than 2/3 of the staked consensus nodes in order to take effect.
If a node is found guilty of committing a slashable protocol violation, 
the consensus nodes directly deduct a fine from the node's stake. 
Of course, only the remaining stake is deposited back into node's balance
at the end of the unstaking period.

Slashing is not handled by the staking contract at this point. 

> It will be handled on a case by case basis. Once full epoch functionality is enabled,

Reviewer: ```suggestion
It will be handled on a case-by-case basis. Once full epoch functionality is enabled,
```
